docs/source/gbd2019_models/risk_exposures/stunting/index.rst

```diff
@@ -0,0 +1,337 @@
+.. role:: underline
+    :class: underline
+
+..
+  Section title decorators for this document:
+
+  ==============
+  Document Title
+  ==============
+
+  Section Level 1 (#.0)
+  +++++++++++++++++++++
+  
+  Section Level 2 (#.#)
+  ---------------------
+
+  Section Level 3 (#.#.#)
+  ~~~~~~~~~~~~~~~~~~~~~~~
+
+  Section Level 4
+  ^^^^^^^^^^^^^^^
+
+  Section Level 5
+  '''''''''''''''
+
+  The depth of each section level is determined by the order in which each
+  decorator is encountered below. If you need an even deeper section level, just
+  choose a new decorator symbol from the list here:
+  https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections
+  And then add it to the list of decorators above.
+
+
+
+.. _2019_risk_exposure_child_stunting:
+
+==============
+Child stunting
+==============
+
+.. contents::
+  :local:
+
++-------------------------------------------------+
+| List of abbreviations                           |
++=======+=========================================+
+| TMREL | theoretical minimum risk exposure level |
++-------+-----------------------------------------+
+| CGF   | child growth failure composed of wasting|
+|       | stunting and underweight                |
++-------+-----------------------------------------+
+| HAZ   | Height for age                          |
++-------+-----------------------------------------+
+| DD    | Diarrheal disease                       |
++-------+-----------------------------------------+
+| LRI   | lower respiratory tract infection       |
++-------+-----------------------------------------+
+| MSLS  | measles                                 |
++-------+-----------------------------------------+
+
+
+.. note:: 
+   
+   Should we call this 2020 and have this under 2020 exposure? How do we deal with the different age groups? 
+
+.. _stunting1.0:
+
+Risk Exposure Overview
+++++++++++++++++++++++
+
+Malnutrition is an imbalance between the body’s needs and its use and intake of nutrients. The imbalance can be caused by poor or lacking diet, poor hygiene, disease states, lack of knowledge, and cultural practices, among others. Underweight, stunting, wasting, obesity, and vitamin and mineral deficiencies are all forms of malnutrition. 
+
+**Chronic malnutrition**, also referred to as stunting ...
```

Comment:
Let's use that recent paper on communication strategies to guide how we talk about these concepts (while always staying compatible with GBD, of course...)